src/cli/cli.cpp

```diff
@@ -6430,19 +6623,70 @@ template <> otError Interpreter::Process<Cmd("routereligible")>(Arg aArgs[])
 
 template <> otError Interpreter::Process<Cmd("routerselectionjitter")>(Arg aArgs[])
 {
+    /**
+     * @cli routerselectionjitter
+     * @code
+     * routerselectionjitter
+     * 120
+     * Done
+     * @endcode
+     * @code
+     * routerselectionjitter 120
+     * Done
+     * @endcode
+     * @cparam routerselectionjitter [@ca{jitter}]
+     * @par
+     * Gets or sets the ROUTER_SELECTION_JITTER value.
+     * @sa otThreadGetRouterSelectionJitter
+     * @sa otThreadSetRouterSelectionJitter
+     */ 
     return ProcessGetSet(aArgs, otThreadGetRouterSelectionJitter, otThreadSetRouterSelectionJitter);
 }
 
 template <> otError Interpreter::Process<Cmd("routerupgradethreshold")>(Arg aArgs[])
 {
+    /**
+     * @cli routerupgradethreshold (get,set)
+     * @code
+     * routerupgradethreshold
+     * 16
+     * Done
+     * @endcode
+     * @code
+     * routerupgradethreshold 16
+     * Done
+     * @endcode
+     * @cparam routerupgradethreshold [@ca{threshold}]
+     * @par
+     * Gets or sets the ROUTER_UPGRADE_THRESHOLD value.
+     * @sa otThreadGetRouterUpgradeThreshold
+     * @sa otThreadSetRouterUpgradeThreshold
+     */
     return ProcessGetSet(aArgs, otThreadGetRouterUpgradeThreshold, otThreadSetRouterUpgradeThreshold);
 }
 
 template <> otError Interpreter::Process<Cmd("childrouterlinks")>(Arg aArgs[])
 {
+    /**
+     * @cli childrouterlinks (get,set)
+     * @code
+     * childrouterlinks
+     * 16
+     * Done
+     * @endcode
+     * @code
+     * childrouterlinks 16
+     * Done
+     * @endcode
+     * @cparam childrouterlinks [@ca{links}]
+     * @par
+     * Gets or sets the MLE_CHILD_ROUTER_LINKS value.
+     * @sa otThreadGetChildRouterLinks
+     * @sa otThreadSetChildRouterLinks
+     */
     return ProcessGetSet(aArgs, otThreadGetChildRouterLinks, otThreadSetChildRouterLinks);
 }
-
+ 
```

Comment:
Remove this blank line... try not to add/remove blank lines that already exist in the file, whatever is already there should be left as is